web/src/store/actionCreators/actionTypes.ts

```diff
@@ -57,3 +57,7 @@ export const FETCH_RUN_FACETS = 'FETCH_RUN_FACETS'
 export const FETCH_JOB_FACETS = 'FETCH_JOB_FACETS'
 export const FETCH_FACETS_SUCCESS = 'FETCH_FACETS_SUCCESS'
 export const RESET_FACETS = 'RESET_FACETS'
+
+// tags
+export const FETCH_TAGS = 'FETCH_TAGS'
+export const FETCH_TAGS_SUCCESS = 'FETCH_TAGS_SUCCESS'
```

Comment:
I think you've got a few newline issues. The project has a formatter that should help you in your IDE.